ios/Sources/UIApplicationClient/Client.swift

```diff
@@ -0,0 +1,19 @@
+import Combine
+import UIKit
+
+public protocol UIApplicationClientProtocol {
+    func open(url: URL, options: [UIApplication.OpenExternalURLOptionsKey: Any]) -> AnyPublisher<Bool, Never>
+}
+
+public struct UIApplicationClient: UIApplicationClientProtocol {
+    public init() {}
+
+    public func open(url: URL, options: [UIApplication.OpenExternalURLOptionsKey: Any]) -> AnyPublisher<Bool, Never> {
+        Future<Bool, Never> { promise in
+            UIApplication.shared.open(url, options: options) { success in
```

Comment:
By showing WebContent in Apps, there are many ways to do.
Is this the way android app does?